### ISSUE TITLE
Error installing Authorize.net Add-on

### ISSUE THREAD

**Issue by craigmarcussen** [concretecms-community-store/community_store_authorize_net#5]
Hello,

First off let me say that I love the Community Store Add-on! I only discovered it a week ago, yet I have already built my first store using the PayPal payment gateway. I am extremely impressed with the depth of features, the intuitive UX, and the extensibility with native Concrete5 Attributes. Simply put, a joy to use.

One issue I am having... when I try to add the **Authorize.net** or the **Stripe** Add-on, I get an error when installing. I have tried installed Composer in the Authorize.net folder, and in the Community Store folder, but neither has helped with the issue.

I am getting this base error:
`Call to a member function getPackageHandle() on null`

The error is calling out this function:
```
protected function setMethodController()
    {
        $th = Application::getFacadeApplication()->make("helper/text");
        $namespace = "Concrete\\Package\\" . $th->camelcase(Package::getByID($this->pkgID)->getPackageHandle()) . "\\Src\\CommunityStore\\Payment\\Methods\\" . $th->camelcase($this->pmHandle);
        $className = $th->camelcase($this->pmHandle) . "PaymentMethod";
        $namespace = $namespace . '\\' . $className;
        $this->methodController = new $namespace();
    }
```

Any suggestions? I don't need to use the Authorize.net gateway for this project, but I am sure I will need it in the near future.

Once again, thank you for a terrific Add-on!

Warmly,
Craig Marcussen

**Comment by Mesuva** [comment]
Hi Craig, thanks for the kind words.

With the version of the Authorize.net package, did you try with the release https://github.com/concrete5-community-store/community_store_authorize_net/releases
Or, was this using the master?

I just tried installing with version 1.0.2 (which is the master branch), and that installed cleanly for me. I'm wondering if this is a case sensitivity issue somewhere... 

Did you end up with anything in the CommunityStorePaymentMethods database table for Authorize.net?

And was this error thrown _on_ install, or when you try to view the different payment methods in the settings dashboard page for the store?

**Comment by craigmarcussen** [comment]
I pulled from the master, instead of the release, and that was the issue. I got a reply on the Concrete5 forum just a few mins ago with the same suggestion (from **mnakalay**) , and It worked perfectly.

The error was thrown on install, and broke the **Payments** area of the Dashboard, among other things. I didn't look into any DB entries. I just reverted to my backup DB.

Again, thank you for such a quick reply, and for such outstanding work!
